lib/core.js

```diff
@@ -1,10 +1,16 @@
-var http = require('http'),
-	moment = require('moment'),
-	_ = require('lodash'),
-	helper = require('./helper'),
-	querystring = require('querystring'),
-	url = require('url'),
-	apierror = require('./error');
+// var http = require('http')
+
+var moment = require('moment')
```

Comment:
how are we using var over here ?
please deploy the eslint widely used in out projects

Reply:
done